src/components/FiltersDrawer/index.tsx

```diff
@@ -23,6 +24,8 @@ const FiltersDrawer: React.FC<FiltersDrawerProps> = ({ "data-cy": dataCy }) => {
     Cookie.get(HAS_OPENED_DRAWER) === "true"
   );
 
+  useKeyboardShortcut(["Control", "d"], () => setCollapsed(!collapsed));
```

Comment:
The FilterDrawer already has a keyboard shortcut by default the `[` key it may be confusing to add 2 different shortcuts.

Reply:
oops sorry I didn't know that